test/jasmine/assets/check_texttemplate.js

```diff
@@ -0,0 +1,28 @@
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+var failTest = require('../assets/fail_test');
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+
+'use strict';
+
+module.exports = function checkTextTemplate(mock, selector, tests) {
```

Comment:
Very nice test!

---

Could you also defaults tests for each of the traces asserting that `textinfo` isn't coerced when `texttemplate` is set?

Reply:
Can I add this test in the `checkTextTemplate` routine to keep the code DRY or should I add a new entry in the appropriate `describe('Test traceName defaults')` block for each traces?

Reply:
> Can I add this test in the `checkTextTemplate` routine to keep the code DRY

Yeah, that would be fine.

Reply:
Done in https://github.com/plotly/plotly.js/commit/9330290d01efb3e24aa2f39b37d2feb34237ab73